trade-finance-manager-api/src/v1/tasks/assign-group-tasks-to-one-user.js

```diff
@@ -57,8 +58,13 @@ const assignGroupTasksToOneUser = async (dealId, groupTitlesToAssign, userId) =>
     },
   };
 
-  await api.updateDeal(dealId, tfmDealUpdate, (status, message) => {
-    throw new Error({ status, message });
+  await api.updateDeal({
+    dealId,
+    dealUpdate: tfmDealUpdate,
+    userInformation: generateTfmUserInformation(sessionTfmUser._id),
```

Comment:
again feel itd be good to generate this at this start -- what do you think? I guess it could have issues if we ever wanted to add new params to the audit obj mid journey?